.github/workflows/main.yml

```diff
@@ -16,11 +15,36 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
 
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDE.md)
+          echo "::set-output name=review_guidelines::$REVIEW_GUIDELINES"
+
       - name: AI Code Review Action
         uses: freeedcom/ai-codereviewer@v2.6.0
         with:
```

Comment:
Please ensure that the `OPENAI_API_KEY` is securely stored and not exposed in the workflow file. It's good practice to store sensitive data like API keys as GitHub secrets.